docs/source/getting-started/index.rst

```diff
@@ -67,53 +68,57 @@ Now, use Yarn to install the project dependencies and build the app::
     yarn install
     yarn workspaces run build
 
-You should see ``Compiled successfully.`` in the output if everything is working as expected.
+You should see ``Compiled successfully.`` in the output if everything is worked expected.
 
 .. TODO: Give instructions for possible failures.
 
 We can now run the app in two steps.
 You'll need two terminal windows running for this.
-
 In one terminal, at the root of the ``create-daml-app`` directory, run the script::
 
     ./daml-start.sh
 
-This compiles the DAML component of the project and starts a *Sandbox* ledger for the app.
-The ledger in this case is stored in the Sandbox application memory; it is not persistent but is useful for testing and development.
-We'll leave the Sandbox running to serve requests from the UI, which result in changes to the in-memory ledger.
+This script is just shorthand for ``daml start`` with some arguments, which does a few things:
+
+    1. Compiles the DAML code to a DAR file as in the previous ``daml build`` step.
+    2. Starts an instance of the :doc:`Sandbox </tools/sandbox>`, an in-memory ledger useful for development, loaded with our DAR.
+    3. Starts a server for the :doc:`HTTP JSON API </json-api/index>`, a simple way to run commands against a DAML ledger (in this case the running Sandbox).
+
+We'll leave these processes running to serve requests from our UI.
 
 In a second terminal, navigate to the ``create-daml-app/ui`` folder and run::
 
     yarn start
 
-This starts the UI application connected to the already running Sandbox.
+This starts the web UI connected to the running Sandbox and JSON API server.
 The command should automatically open a window in your default browser at http://localhost:3000.
-If it doesn't, just open that link in any web browser.
+If it doesn't, just open that link in a web browser.
 (Depending on your firewall settings, you may be asked whether to allow the app to receive network connections. It is safe to accept.)
 
-Now you should see the login page for the social network.
-For simplicity of this app, there is no password required to sign up or log in.
-To learn how to handle authentication, see this blog post about `DAML and Auth0 <https://blog.daml.com/daml-driven/easy-authentication-for-your-distributed-app-with-daml-and-auth0>`_ or the :doc:`full documentation </app-dev/authentication>`.
+You should now see the login page for the social network.
+For simplicity of this app, there is no password or sign-up required.
+To learn how to handle proper authentication, see this blog post about `DAML and Auth0 <https://blog.daml.com/daml-driven/easy-authentication-for-your-distributed-app-with-daml-and-auth0>`_ or the :doc:`full documentation </app-dev/authentication>`.
 
-First enter your name and click "Sign up".
+First enter your name and click "Log in".
```

Comment:
I don’t recommend putting things in quote that you want to highlight. Either make them italic (for text) or turn them into inline code using double backticks.

Reply:
Ok. I was using italics for new terms and concepts, backticks for code and double quotes for elements of the UI. But I can use italics for that as well